src/test/mod.rs

```diff
@@ -52,6 +53,18 @@ impl Test {
         }
     }
 
+    pub fn from_missed_words(missed_words: &Vec<String>) -> Self {
```

Comment:
The `test` module is only concerned with handling test key events and generating results, not creating test contents. The code to create the word list should be in `main.rs`, like `Opt::gen_contents`.

Reply:
made this update